package.json

```diff
@@ -10,14 +10,16 @@
     "datadog",
     "DogStatsD"
   ],
-  "version": "0.14.2",
+  "version": "0.15.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/ysa23/metrics-reporter"
   },
   "scripts": {
     "test": "jest --runInBand --forceExit --detectOpenHandles",
     "lint": "eslint src/index.js .",
+    "build-declarations": "tsc",
+    "watch-declarations": "tsc -w",
```

Comment:
you can do "declarations:build" and "declarations:watch" . but maybe it's better to have a shorter name, like "tsc:build" "tsc:watch"